### PR TITLE
Apply whiterabbit design

### LINKED ISSUE #46: Apply WhiteRabbit designs to the Movie Page
Mockups from Alan for guidance (not to be copied as is):
- [WR Plug-in sitemap](https://user-images.githubusercontent.com/163447/79341394-6784ba00-7f34-11ea-8a4a-caf82ed19b08.png)
- [Plug in presentation.pdf](https://github.com/leapdao/mad-hatter/files/4481355/Plug.in.presentation.pdf)

Resources:
- [White-rabbit-logo](https://user-images.githubusercontent.com/163447/79374863-64072800-7f60-11ea-8d21-7464918ca19a.png)
- [White-rabbit-logo-white](https://user-images.githubusercontent.com/163447/79374868-65d0eb80-7f60-11ea-99d2-cf43b7c093d0.png)
- [White rabbit logo.pdf](https://github.com/leapdao/mad-hatter/files/4483063/White.rabbit.logo.pdf)
- [White rabbit logo.eps.zip](https://github.com/leapdao/mad-hatter/files/4483068/White.rabbit.logo.eps.zip)

## Scope
- General styles
  - [x] apply colors and fonts
  - [x] change header according to attached screen. Keep wallet address available under "user" icon
- Movie Page
  - [x] make poster full bleed
  - [x] adjust styles for text, buttons etc. Keep "decline" button, keep disabled button styling we have
  - [x] combine crew and credits details together (see screenshot)
  - [x] show IMDB rating
  - [x] show movie year
- Acknowledgement Page
  - [x] make poster full bleed. Keep the page structure (contrary to the mockup)

## Roles

gardener: @troggy 50 DAI
worker: @mwaa  400 DAI
reviewer: @Clearwood 100 DAI

## Screens

image

### PR DESCRIPTION
**Pull request by mwaa**
Please remember the [Contributing Guidelines](https://github.com/leapdao/meta/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

- [x] Full page background posters
- [x] IMDB rating and release year
- [x] Change of font to gt-cinetype

## Recommendations for testing

Clone and test locally

## Links to relevant issues or information

Closes #46

### CODE REVIEW

**Comment by Clearwood** [comment]
It looks pretty good, good work first of all :+1: I saw some problems with the appreciation site and overflow and margin issues with the general wallet pages. Additionally there is no way to navigate back from the account page to the movie page. I also think we should remove the menu icon until we have a use for it. 
The button to scan a QR-code does not work anymore. 
github_email pl

**Comment by mwaa** [comment]
@Clearwood i can address the success page appreciation page errors

@troggy  Need some feedback on whether to address these now or these will be addressed
while working on the other white-rabbit pages from the plugin design document.

1. Removal of menu icon until it has functionality
2. Navigation back to movie page from accounts page
3. Wallet margin issues on none white-rabbit pages
4. The button to scan a QR-code does not work anymore

**Comment by Clearwood** [comment]
> @Clearwood i can address the success page appreciation page errors
> 
> @troggy Need some feedback on whether to address these now or these will be addressed
> while working on the other white-rabbit pages from the plugin design document.
> 
> 1. Removal of menu icon until it has functionality
> 2. Navigation back to movie page from accounts page
> 3. Wallet margin issues on none white-rabbit pages
> 4. The button to scan a QR-code does not work anymore

Awesome summary, sounds great, the appreciation page is def the most important

**Comment by troggy** [comment]
> Removal of menu icon until it has functionality
Let's hide it for now

> Navigation back to movie page from accounts page
Out of scope. This flow wasn't thought through and will be probable hidden in the final app, so I suggest not to spend much time on it.

> Wallet margin issues on none white-rabbit pages
Out of scope, unless it is a quick fix. If there is quick way to fix that, it should be done. Otherwise, not a priority as well — we don't know yet which of the screens we will eventually need.

> The button to scan a QR-code does not work anymore
Out of scope, unless it is a quick fix. I don't think we will need scanner in the final product

File: packages/burner-plugin/src/ui/MoviePage.tsx [hunk 1]
```diff
@@ -134,13 +128,7 @@ const MoviePage: React.FC<MoviePageContext> = ({
...
     <Page>
       <ImageBackdrop backgroundImage={movieData.posterUrl}>
-        <MovieDetailsPanel>
-          <h1>{movieData.title}</h1>
-          <LineItem name={'Director:'} value={movieData.director}/>
-          <LineItem name={'Producer:'} value={movieData.producer}/>
-          <LineItem name={'Actors:'} value={movieData.actors.join(',  ')}/>
-          <LineItem name={'Production company:'} value={movieData.productionCompanies.join(', ')}/>
-        </MovieDetailsPanel>      
+        <MovieDescription movieData={movieData}></MovieDescription>
```
Comment (troggy) [inline packages/burner-plugin/src/ui/MoviePage.tsx:131]: ❤️

**Pull request merged**